src/features/notifications/NotificationsScreen.tsx

```diff
@@ -1,23 +1,48 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import { useSelector } from 'react-redux'
 import SafeAreaBox from '../../components/SafeAreaBox'
+import { fetchNotifications } from '../../store/account/accountSlice'
 import { RootState } from '../../store/rootReducer'
-import NoNotifications from './NoNotifications'
+import { useAppDispatch } from '../../store/store'
+import useVisible from '../../utils/useVisible'
+import { postWallet } from '../../utils/walletClient'
+import EmptyNotifications from './EmptyNotifications'
 import NotificationList from './NotificationList'
 
 const NotificationsScreen = () => {
   const {
-    account: { notifications },
+    account: { notifications, markNotificationStatus },
   } = useSelector((state: RootState) => state)
+  const dispatch = useAppDispatch()
+
+  const markAsRead = useCallback(() => {
+    if (!notifications.find((n) => !n.viewed_at)) return
```

Comment:
I noticed that I had to reload the app to get the `new` indicator to disappear from my notifications

Reply:
Interesting. I'll try to reproduce.

Reply:
if it's not reproducible it could have just been something weird with my dev env